pom.xml

```diff
@@ -36,7 +36,7 @@
                         </goals>
                         <!-- Optional configuration which provides for running any npm command -->
                         <configuration>
-                            <arguments>install --production</arguments>
+                            <arguments>ci --only=production</arguments>
```

Comment:
use npm ci for jitpack as well (jitpack is actually for java, which is why we are using a pom file)